client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -48,28 +48,28 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 	);
 	const previousProgressValue = stepProgress ? previousProgress / stepProgress.count : 0;
 
-	const stepNavigation = flow.useStepNavigation(
-		currentStepRoute,
-		async ( path, extraData = null ) => {
-			// If any extra data is passed to the navigate() function, store it to the stepper-internal store.
-			setStepData( {
-				path: path,
-				intent: intent,
-				...extraData,
-			} );
-
-			const _path = path.includes( '?' ) // does path contain search params
-				? generatePath( `/${ flow.name }/${ path }` )
-				: generatePath( `/${ flow.name }/${ path }${ search }` );
-
-			history.push( _path, stepPaths );
-			setPreviousProgress( stepProgress?.progress ?? 0 );
-		}
-	);
+	const navigate = async ( path: string, extraData = {} ) => {
```

Comment:
Nothing changed. Just declare it first and pass it to the `useStepNavigation` and `useSideEffect` hook